src/mono/wasm/runtime/types/index.ts

```diff
@@ -93,6 +93,16 @@ export type MonoConfig = {
      * If true, the snapshot of runtime's memory will be stored in the browser and used for faster startup next time. Default is false.
      */
     startupMemoryCache?: boolean,
+    /**
+     * If true, a list of the methods optimized by the interpreter will be saved and used for faster startup
+     *  on future runs of the application
+     */
+    interpreterPgo?: boolean,
+    /**
+     * Configures how long to wait before saving the interpreter PGO list. If your application takes
+     *  a while to start you should adjust this value.
+     */
+    interpreterPgoSaveDelay?: number,
```

Comment:
please run `npm run rollup` or msbuild in debug mode to re-generate and update `dotnet.d.ts`. Don't do it by hands, please.